assignments/objects.js

```diff
@@ -19,27 +19,54 @@ const example = {
 
 // Write your intern objects here:
 
+function intern(id, name, email, gender) {
+  return {id, name, email, gender};
+}
+
+const Mitzi = intern(1, 'Mitzi', 'mmelloy0@psu.edu', 'F');
+const Kennan = intern(2, 'Kennan', 'kdiben1@tinypic.com', 'M');
+const Keven = intern(3, 'Keven', 'kmummery2@wikimedia.org', 'M');
+const Gannie = intern(4, 'Gannie', 'gmartinson3@illinois.edu', 'M');
+const Antonietta = intern(5, 'Antonietta', 'adaine5@samsung.com', 'F');
```

Comment:
While you won't have any syntax problems with your variable naming, it's considered best practice to only capitalize classes in Javascript, variables and should be lowercase :+1: